src/pages/index.js

```diff
@@ -11,166 +11,125 @@ import {
   useTranslation,
   Trans,
 } from '@newrelic/gatsby-theme-newrelic';
-import { rgba } from 'polished';
 import SurfaceLink from '../components/SurfaceLink';
 import TechTile from '../components/TechTile';
 import TechTileGrid from '../components/TechTileGrid';
-import NetworkPerformanceMonitoringBannerGA from '../components/NetworkPerformanceMonitoringBannerGA';
+import HomepageBanner from '../components/HomepageBanner';
 import {
   tdp,
   fso,
   ai,
   security,
   integrations,
   mobile_apps,
+  getting_started,
+  popular_content,
 } from '../data/homepage.yml';
 
 const HomePage = ({ data }) => {
   const {
     site: { layout },
+    allMarkdownRemark: { edges: whatsNewPosts },
   } = data;
 
   const { t } = useTranslation();
 
+  const latestWhatsNewPosts = whatsNewPosts.slice(0, 3).map((edge) => {
```

Comment:
If we only need three, then we can add a `limit` to the graphql query, which should speed up the time it takes to build this page. I'll comment a suggestion down below too.

```suggestion
  const latestWhatsNewPosts = whatsNewPosts.map((edge) => {
```

Reply:
oh perfect!